circle.yml

```diff
@@ -26,6 +26,7 @@ dependencies:
   override:
     - ./scripts/time-cmd.sh ./gradlew --profile --parallel resolveConfigurations
     - scripts/circle-ci/pr-changelog-status-check.sh
+    - ./gradlew --stacktrace generateLock verifyDependencyLocksAreCurrent
```

Comment:
I don't see the need for --stacktrace here.  Also I imagine it's worth including this in gradle check instead of having this additional check here.